presto-docs/src/main/sphinx/functions/math.rst

```diff
@@ -182,3 +182,8 @@ Floating Point Functions
 .. function:: nan() -> double
 
     Returns the constant representing not-a-number.
+
+.. function:: width_bucket(x, bound1, bound2, n) -> bigint
```

Comment:
The standard specifies that the upper and lower bounds can be either the first or second bound. That's why I named them this way. Should I still rename them?